src/App.js

```diff
@@ -1,12 +1,16 @@
 import { Routes, Route } from "react-router-dom";
 import Home from "./Pages/Home";
```

Comment:
You can do a barrel import / export for all you components / files. Makes the code cleaner